pkg/api/types.go

```diff
@@ -1205,6 +1208,21 @@ type KeyToPath struct {
 	Mode *int32
 }
 
+// LocalVolume represents a local volume accessible only from one node
+type LocalVolumeSource struct {
+	// NodeName represents the node that the volume is located at
+	NodeName string
```

Comment:
Actually I think this would be better as an annotation in the PV, then we can use it for other plugins too like gcepd.  

For local volumes I would still want to require that the topology annotation is set (specific key values are not validated), so that we can ensure that we're mounting the paths on the correct node.